services/queue/src/artifacts.js

```diff
@@ -439,18 +446,12 @@ builder.declare({
     '`anonymous` role.  The convention is to include',
     '`queue:get-artifact:public/*`.',
     '',
-    '**API Clients**, this method will redirect you to the artifact, if it is',
-    'stored externally. Either way, the response may not be JSON. So API',
-    'client users might want to generate a signed URL for this end-point and',
-    'use that URL with an HTTP client that can handle responses correctly.',
-    '',
-    '**Downloading artifacts**',
-    'There are some special considerations for those http clients which download',
-    'artifacts.  This api endpoint is designed to be compatible with an HTTP 1.1',
-    'compliant client, but has extra features to ensure the download is valid.',
-    'It is strongly recommend that consumers use either taskcluster-lib-artifact (JS),',
-    'taskcluster-lib-artifact-go (Go) or the CLI written in Go to interact with',
-    'artifacts.',
+    '**Response**: the HTTP response to this method is a 303 redirect to the',
+    'URL from which the artifact can be downloaded.  The body of that response',
+    'contains the data described in the output schema, contianing the same URL.',
+    'Callers are encouraged to use whichever method of gathering the URL is',
+    'most convenient.  Standard HTTP clients will follow the redirect, while',
+    'API  client libraries will return the JSON body.',
```

Comment:
```suggestion
    'API client libraries will return the JSON body.',
```

Reply:
This will require `yarn generate` so I'll apply it locally and push again.